### PR TITLE
Update paragraph on install-required-packages

### PR DESCRIPTION
**Pull request by prologic**
##### Summary

This tool is very much __NOT__ experimental, so updating it to reflect current reality.

##### Component Name

- area/docs

##### Test Plan

- Netlify

##### Additional Information

### CODE REVIEW

**Comment by prologic** [comment]
> 2:02:10 PM:    - ERROR: ./packaging/installer/methods/manual.md - https://github.com/netdata/netdata/blob/master/packaging/installer/install-required-packages.sh is an absolute link to a Netdata file. Please convert to relative.

Hmmm 🤔 @joelhans Is this a new check? I'm open to ideas or just remove this bit. I originally only wanted to remove the wording "experimental".

**Comment by joelhans** [comment]
@prologic This isn't a new check, no. The easiest fix that works for me locally is to just link to the raw version of the file: `https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh`. That way it doesn't get tripped up in the build script. 

I would say do that fix, or don't link it at all. There's another solution that involves linking to a specific blob, but that means if you update the file the link will reference an old version, which is not good.

File: packaging/installer/methods/manual.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ To install the latest git version of Netdata, please follow these 2 steps:
 
 ## Prepare your system
 
-Try our experimental automatic requirements installer (no need to be root). This will try to find the packages that
-should be installed on your system to build and run Netdata. It supports most major Linux distributions released after
-2010:
+Use our automatic requirements installer (_no need to be `root`_). This will try to find the packages that
```
Comment (shortpatti) [inline packaging/installer/methods/manual.md]:
```suggestion
Use our automatic requirements installer (_no need to be `root`_), which attempts to find the packages that
```

**Comment by prologic** [comment]
@shortpatti Oh I _think_ we both tried to fix the build error at the same time? :) Thanks! If this works please review/approve so we can merge this

**Pull request merged**